layouts/shortcodes/wordlimit.html

```diff
@@ -1,10 +1,10 @@
-<section>
+<section id="wordlimit">
```

Comment:
If this is a shortcode, presumably it can be called on a page mulitple times. In which case, please uniquify the id.

Secondly, please add a BEM class and put your styles on the class not the id. TY!